docs/schema-loader.md

```diff
@@ -4,9 +4,9 @@ This document briefly explains tools used to generate and load database schemas
 
 ## Scalar DB schema generator and loader
 
-Scalar DB schema generator (tools/schema/generator) generates storage implementation specific schema definition and metadata definition for Scalar DB transaction
```

Comment:
The whole doc can be removed.
Please check the pages that refer to the doc and update them to refer to the schema tool doc.

Reply:
Thank you for the review comments.
Document updated accordingly.